src/components/InputForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import styled from "styled-components";
+
+import { IdentityTypes } from "../@types/Identity";
+
+export const InputForm: React.FC<{ type: IdentityTypes }> = ({ type }) => {
+  const [formValue, setFormValue] = React.useState("");
```

Comment:
This is more of a `EmailIdentityForm` with `const [email, setEmail] = React.useState("")` don't you think?

Reply:
Yeah true, I wanted to have a global component but it's not needed there. I'll change it to make it more readable.